src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,61 @@
+package lotto.domain;
+
+import java.math.BigDecimal;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class Lotto {
+    public static final BigDecimal PRICE = BigDecimal.valueOf(1000);
+    public static final int NUMBER_OF_LOTTO = 6;
+    public static final int LOTTO_START_NUM = 1;
+    public static final int LOTTO_END_NUM = 45;
+
+    private Set<LottoNumber> lotto;
+
+    public Lotto(List<LottoNumber> lotto) {
+        this.lotto = new HashSet<LottoNumber>(lotto);
```

Comment:
이 경우엔 타입을 생략해도 무방합니다 :)
```suggestion
        this.lotto = new HashSet<>(lotto);
```